examples/AdvancedConsumer/AdvancedConsumer.csproj

```diff
@@ -10,4 +10,10 @@
     <ProjectReference Include="../../src/Confluent.Kafka/Confluent.Kafka.csproj" />
   </ItemGroup>
 
```

Comment:
windows platform packages are transitively linked via Confluent.Kafka (x86 and x64). Any other platform requires an additional package reference.